slave-base/contrib/bin/run-jnlp-client

```diff
@@ -29,6 +29,9 @@ if [[ "$(uname -m)" == "x86_64" ]]; then
       alternatives --set java $(alternatives --display java | awk '/family.*i386/ { print $1; }')
       export MALLOC_ARENA_MAX=${MALLOC_ARENA_MAX:-1}
     fi
+#set JVM for all other archs
```

Comment:
similar update to the comment in line 21 to what I asked for in the main image run script

Reply:
ack. new commit coming soon.